src/os_maild/os_maild_client.c

```diff
@@ -378,77 +379,18 @@ MailMsg *OS_RecvMailQ_JSON(file_queue *fileq, MailConfig *Mail, MailMsg **msg_sm
     if (!(rule = cJSON_GetObjectItem(al_json, "rule"), rule && (mail_flag = cJSON_GetObjectItem(rule, "mail"), mail_flag && cJSON_IsTrue(mail_flag))))
         goto end;
 
+
     /* If e-mail came correctly, generate the e-mail body/subject */
     os_calloc(1, sizeof(MailMsg), mail);
     os_calloc(BODY_SIZE, sizeof(char), mail->body);
     os_calloc(SUBJECT_SIZE, sizeof(char), mail->subject);
 
+    // Add alert to logs
+    char *tab = malloc(14*sizeof(char));
+    strcpy(tab, "\t");
+    cJSON_PrintTable(al_json, logs, body_size, tab, 2);
+    free(tab);
 
-    // Get full_log field content
-    json_field = cJSON_GetObjectItem(al_json,"full_log");
-    if (!json_field) {
-        goto end;
-    }
-
-    log_size = strlen(json_field->valuestring) + 4;
-
-    /* If size left is small than the size of the log, stop it */
-    if (body_size <= log_size) {
-        goto end;
-    }
-
-    strncpy(logs, json_field->valuestring, body_size);
-    strncpy(logs + log_size, "\r\n", body_size - log_size);
-    body_size -= log_size;
-
-
-    json_object = cJSON_GetObjectItem(al_json,"syscheck");
```

Comment:
The FIM alert should be printed separately as was being done. This is why the alert doesn't specify which parameters of the modified file have been modified.

Reply:
I resolve the conversation because we've already agreed that it won't be done.